packages/memory/src/types.ts

```diff
@@ -30,7 +30,7 @@ export type IFsMemNodeType = IFsMemFileNode | IFsMemDirectoryNode | IFsMemSymlin
 
 export interface IFsMemFileNode extends IFsMemNode {
   type: "file";
-  contents: string;
+  contents: Uint8Array;
```

Comment:
How about storing `string | Uint8Array` and then converting only if necessary in readFile / readFileSync?

Reply:
I did it but I'm not sure why it's not merged. #804